audio/sdl2-audio/source/main.c

```diff
@@ -0,0 +1,71 @@
+// Include the most common headers from the C standard library
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+// Include the main libnx system header, for Switch development
+#include <switch.h>
+
+// Include sdl2 headers
+#include <SDL2/SDL.h>
+#include <SDL2/SDL_mixer.h>
+
+// Main program entrypoint
+int main(int argc, char* argv[])
+{
+    // This example uses sdl2 library to play a mp3 file
+	
+    consoleInit(NULL);
+
+
+    Result rc = romfsInit();
+    if (R_FAILED(rc))
+    	printf("romfsInit: %08X\n", rc);
+	
+	else
+		printf("Press A button to play the sound!\n");
+
+    // Start SDL with audio support
+    SDL_Init(SDL_INIT_AUDIO);
+	 
+    // Load support for the MP3 format 
+    Mix_Init(MIX_INIT_MP3);
+	 
+    // open 44.1KHz, signed 16bit, system byte order,
+    //  stereo audio, using 4096 byte chunks
+    Mix_OpenAudio(44100, MIX_DEFAULT_FORMAT, MIX_DEFAULT_CHANNELS, 4096);
+	
+    // Load sound file to use
+	//Sound from https://freesound.org/people/jens.enk/sounds/434610/
```

Comment:
tab -> 4 spaces.

Reply:
I know, I'm using Notepad++ where the spaces appear correct, I'd better switch to visual code